Hydrogen/phi= 0.1/Asaba_1965_ST_fig_6_fcircle.yaml

```diff
@@ -0,0 +1,138 @@
+    authors: 
+        - name: T. Asaba
+        - name: W.C.Jr. Gardiner
+        - name: R.F. Stubbeman
+    journal: Symposium (International) on Combustion
+    year: 1965
+    volume: 10
+    pages: 295-302
+    detail: Fig.6. - full circle [Converted from ReSpecTh XML file x10000027_x.xml]
+experiment-type: ignition delay
+apparatus:
+    kind: shock tube
+    institution: University of Texas, Austin TX
+    facility: stainless steel shock tube
+common-properties:
+    composition: &comp
+      kind: mole fraction
+      species:
+        - species-name: H2
+          InChI: 1S/H2/h1H
+          amount:
+            - 0.01
+        - species-name: O2
+          InChI: 1S/O2/c1-2
+          amount:
+            - 0.03
+        - species-name: Ar
+          InChI: 1S/Ar
+          amount:
+            - 0.96
+    ignition-type: &ign
+        target: OH
+        type: max 
+datapoints:
+    - temperature:
+        - 2258 K
+      ignition-delay:
+        - 18.6 us
+      pressure:
+        - 0.399 atm
+      composition: *comp
+      ignition-type: *ign 
+    - temperature:
+        - 2278 K
+      ignition-delay:
+        - 26.7 us
+      pressure:
+        - 0.403 atm
+      composition: *comp
+      ignition-type: *ign 
+    - temperature:
+        - 2320 K
+      ignition-delay:
+        - 29.9 us
+      pressure:
+        - 0.412 atm
+      composition: *comp
+      ignition-type: *ign 
+    - temperature:
+        - 2074 K
+      ignition-delay:
+        - 49.7 us
+      pressure:
+        - 0.362 atm
+      composition: *comp
+      ignition-type: *ign 
+    - temperature:
+        - 1929 K
+      ignition-delay:
+        - 34.3 us
+      pressure:
+        - 0.334 atm
+      composition: *comp
+      ignition-type: *ign 
+    - temperature:
+        - 1838 K
+      ignition-delay:
+        - 63.7 us
+      pressure:
+        - 0.315 atm
+      composition: *comp
+      ignition-type: *ign 
+    - temperature:
+        - 1736 K
+      ignition-delay:
+        - 74 us
+      pressure:
+        - 0.295 atm
+      composition: *comp
+      ignition-type: *ign 
+    - temperature:
+        - 1745 K
+      ignition-delay:
+        - 93.3 us
+      pressure:
+        - 0.297 atm
+      composition: *comp
+      ignition-type: *ign 
+    - temperature:
+        - 1572 K
+      ignition-delay:
+        - 111.7 us
+      pressure:
+        - 0.262 atm
+      composition: *comp
+      ignition-type: *ign 
+    - temperature:
+        - 1604 K
+      ignition-delay:
+        - 144.8 us
+      pressure:
+        - 0.269 atm
+      composition: *comp
+      ignition-type: *ign 
+    - temperature:
+        - 1605 K
+      ignition-delay:
+        - 164.4 us
+      pressure:
+        - 0.269 atm
+      composition: *comp
+      ignition-type: *ign 
+    - temperature:
+        - 1428 K
+      ignition-delay:
+        - 215.7 us
+      pressure:
+        - 0.233 atm
+      composition: *comp
+      ignition-type: *ign 
```

Comment:
Please make sure to add newlines to the end of each file

Reply:
Got it! I will modify the files to include the endline.